### PR TITLE
Perform drag and drop from the browser end

### PR DESCRIPTION
**Pull request by bbondy**
There was a race condition where drop info was not determined on dragEnd. This gets rid of it by moving the handling in the browser process like it should be anyway.

https://github.com/brave/browser-laptop/issues/8288

### CODE REVIEW

**Comment by bbondy** [comment]
@bridiver pls review but I'm merging for now so we can get builds.

File: js/dnd.js [hunk 1]
```diff
@@ -47,9 +29,25 @@ module.exports.onDragStart = (dragType, data, e) => {
   appActions.dragStarted(currentWindowId, dragType, data)
 }
 
+document.addEventListener('keyup', (e) => {
+  if (e.keyCode === 27) {
```
Comment (bsclifton) [inline js/dnd.js]: Can this use the constant for ESC? see `app/common/constants/keyCodes.js`

Comment (luixxiul) [inline js/dnd.js]: @bbondy ^

Comment (bbondy) [inline js/dnd.js]: yep sorry I meant to get back to that before submitting the PR, will do!

**Pull request merged**